core/domain/wipeout_service_test.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for voiceover services."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.domain import rights_manager
+from core.domain import topic_domain
+from core.domain import topic_services
+from core.domain import user_services
+from core.domain import wipeout_service
+from core.platform import models
+from core.tests import test_utils
+import feconf
+
+(user_models,) = models.Registry.import_models([models.NAMES.user])
+
+
+class WipeoutServiceTests(test_utils.GenericTestBase):
```

Comment:
Make sure that, when you write tests, you check all the possible behaviours. For example, there is a line in the corresponding code module saying "this should disable all emails to users" but none of this actually seems to be tested. The test should illustrate: "before making the call, this happens; after making the call, this other thing happens".

In general, please make sure tests are fully comprehensive with regards to behaviour, not just testing to get 100% coverage.